index.js

```diff
@@ -53,16 +57,33 @@ class Hypersign {
   }
 
   signable (value, opts = {}) {
-    const { salt } = opts
+    const { salt, seq = 0 } = opts
     assert(Buffer.isBuffer(value), 'Value must be a buffer')
     assert(value.length <= VALUE_MAX_SIZE, `Value size must be <= ${VALUE_MAX_SIZE}`)
-    if (!salt) return value
-    assert(Buffer.isBuffer(salt), 'salt must be a buffer')
-    assert(
-      salt.length >= 16 && salt.length <= 64,
-      'salt size must be between 16 and 64 bytes (inclusive)'
-    )
-    return Buffer.concat([Buffer.from([salt.length]), salt, value])
+    if (salt) {
+      assert(Buffer.isBuffer(salt), 'salt must be a buffer')
+      assert(
+        salt.length <= 64,
+        'salt size must be no greater than 64 bytes'
+      )
+    }
+
+    return salt ? Buffer.concat([
+      saltSeg,
+      Buffer.from(`${salt.length}:`),
+      salt,
+      seqSeg,
+      Buffer.from(seq.toString()),
```

Comment:
this needs the i e wrap right?

Reply:
the `i` is here: https://github.com/hyperswarm/hypersign/pull/1/files#diff-168726dbe96b3ce427e7fedce31bb0bcR18

I just added the `e` here: https://github.com/hyperswarm/hypersign/pull/1/files#diff-168726dbe96b3ce427e7fedce31bb0bcR19